### PR TITLE
site: add missing `alt` attributes to `img` elements

### LINKED ISSUE #357: site: add missing `alt` attributes

### PR DESCRIPTION
**Pull request by Udassi-Pawan**
I added appropriate alt attributes to images with https links while for local images, I added the name of respective image to the alt prop.
I am relatively newer to open source contributions, thus if there is any change to be made, do let me know.

Fixes #357

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img src="https://raw.githubusercontent.com/GoogleChromeLabs/quicklink/HEAD/assets/images/logos/banner-white-bg.png" alt="" width="640">
+  <img src="https://raw.githubusercontent.com/GoogleChromeLabs/quicklink/HEAD/assets/images/logos/banner-white-bg.png" alt="quicklink logo" width="640">
```
Comment (XhmikosR) [inline README.md:2]: This is presentational, it can be left empty as it was

File: site/src/_includes/components/download.njk [hunk 2]
```diff
@@ -3,11 +3,11 @@
 {% sectionTitle "Download" %}
 <ul class="no-bullet">
   <li class="list-icon">
-    <img src="/assets/images/font-icons/download-solid.svg" alt="download" loading="lazy" width="13" height="13">
+    <img src="/assets/images/font-icons/download-solid.svg" alt="download" loading="lazy" width="13" height="13" alt="download icon">
```
Comment (XhmikosR) [inline site/src/_includes/components/download.njk:6]: This and the next one already have an `alt` attribute so the changes are not needed.

File: translations/zh-cn/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img src="https://i.imgur.com/NVRZLHv.png" width="640" alt="quicklink">
+  <img src="https://i.imgur.com/NVRZLHv.png" width="640" alt="quicklink" alt="chain">
```
Comment (XhmikosR) [inline translations/zh-cn/README.md:2]: Same here, it's already present

**Review by XhmikosR — CHANGES_REQUESTED** [review]

**Comment by XhmikosR** [comment]
The rest of the `alt` changes need to properly describe what the image is showing, not just the filename.

**Comment by Udassi-Pawan** [comment]
Hey @XhmikosR  Thanks for your input and patience, I have made the relevant modifications to the code. Do let me know if any other changes are reqiured.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img src="https://raw.githubusercontent.com/GoogleChromeLabs/quicklink/HEAD/assets/images/logos/banner-white-bg.png" alt="" width="640">
+  <img src="https://raw.githubusercontent.com/GoogleChromeLabs/quicklink/HEAD/assets/images/logos/banner-white-bg.png" width="640">
```
Comment (XhmikosR) [inline README.md:2]:
```suggestion
  <img src="https://raw.githubusercontent.com/GoogleChromeLabs/quicklink/HEAD/assets/images/logos/banner-white-bg.png" alt="" width="640">
```

**Comment by XhmikosR** [comment]
@Udassi-Pawan thanks for the progress. Could you start the alt text with a capital letter please?

Also, I think we have two more instances:

* https://github.com/GoogleChromeLabs/quicklink/blob/7e52414e83290c91e34465e6fd19418be676a313/site/src/approach.njk#L17
* https://github.com/GoogleChromeLabs/quicklink/blob/7e52414e83290c91e34465e6fd19418be676a313/site/src/approach.njk#L26

**Comment by Udassi-Pawan** [comment]
Are any other changes required?

**Review by addyosmani — APPROVED** [review]
LGTM

**Pull request merged**